budvm/src/lib.rs

```diff
@@ -2651,6 +2685,8 @@ macro_rules! checked_op {
                         ))
                     }
                 };
+                // TODO handle failures
+                self.stack.pop_n(to_pop);
```

Comment:
This function can exit on errors early, do we need to clean up the stack in these cases?